Documentation/help_files/openemr_installation_help.php

```diff
@@ -15,10 +15,10 @@
 <html>
     <head>
         <link rel=stylesheet href="../../public/themes/style_light.css">
-        <link rel="stylesheet" href="../../public/assets/jquery-ui/jquery-ui.css" type="text/css">
+        <link rel="stylesheet" href="../../public/assets/jquery-ui/jquery-ui.css">
         <script type="text/javascript" src="../../public/assets/jquery/dist/jquery.min.js"></script>
```

Comment:
It looks like `openemr_installation_help.php` page still contains jQuery-UI, I will remove it.